src/main/java/xyz/aadev/generitech/common/container/power/ContanierGenerator.java

```diff
@@ -19,8 +19,10 @@
 
 import net.minecraft.entity.player.InventoryPlayer;
 import net.minecraft.inventory.IInventory;
+import net.minecraft.inventory.SlotFurnaceFuel;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'net.minecraft.inventory.SlotFurnaceFuel'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.aadev.xyz/coding_rules#rule_key=squid%3AUselessImportCheck)